app/controllers/comments/reactions_controller.rb

```diff
@@ -1,14 +1,7 @@
 # frozen_string_literal: true
 
-class CommentController < ApplicationController
-  def index
-    answer = Answer.find(params[:id])
-    @comments = Comment.where(answer:).includes([{ user: :profile }, :smiles])
-
-    render "index", locals: { a: answer }
-  end
-
-  def show_reactions
+class Comments::ReactionsController < ApplicationController
+  def show
```

Comment:
Think it makes more sense for this to be `index` as it's listing the reactions on a comment rather than showing one